src/core/lib/security/credentials/tls/grpc_tls_credentials_options.cc

```diff
@@ -29,6 +29,44 @@
 
 #include "src/core/lib/surface/api_trace.h"
 
+int grpc_tls_credentials_options::cmp(
+    const grpc_tls_credentials_options* other_options) const {
```

Comment:
This parameter can be a reference instead of a pointer.

Reply:
Sure. The security connector's cmp() function takes a pointer though...do we want to make this consistent?